packages/iov-bns-governance/src/integrationtests/proposals.spec.ts

```diff
@@ -36,8 +36,11 @@ import { CommitteeId, ProposalOptions, ProposalType } from "../proposals";
 const adminMnemonic = "degree tackle suggest window test behind mesh extra cover prepare oak script";
 const adminPath = HdPaths.iov(0);
 const bnsdUrl = "ws://localhost:23456";
-const guaranteeFundEscrowId = Encoding.fromHex("0000000000000001");
-const guaranteeFundAddress = escrowIdToAddress("local-iov-devnet" as ChainId, guaranteeFundEscrowId);
+const guaranteeFundEscrowId = 1;
+const guaranteeFundAddress = escrowIdToAddress(
+  "local-iov-devnet" as ChainId,
+  Encoding.fromHex("0000000000000001"),
+);
```

Comment:
Can we change the input type of escrowIdToAddress too please? I added this a couple of days ago to the ticket